recipes/wgs-assembler/meta.yaml

```diff
@@ -15,8 +15,10 @@ source:
   md5: 70e8efea279d7061718d442a8c1217b3
 
 build:
-  number: 2
+  number: 3
   skip: True # [py3k or osx]
```

Comment:
The recipe is skipped for Python 3.x.
This is the reason why all builds passed without producing any artifacts.